src/modules/jcms_admin/jcms_admin.module

```diff
@@ -193,3 +250,55 @@ function _jcms_admin_node_title_unique($title, $node, $content_type = NULL) {
 
   return TRUE;
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ */
+function jcms_admin_entity_subqueue_presave(EntityInterface $entity) {
+  if ($entity->bundle() != 'covers_preview') {
+    return NULL;
+  }
+  if (_jcms_admin_covers_active('publish')) {
+    $items = ($entity->get('items')) ? $entity->get('items')->getValue() : [];
+    $items = array_splice($items, 0, $entity->get('field_covers_active_items')->getString());
+    foreach ($items as $item) {
+      $node = Node::load($item['target_id']);
+      $moderation_info = \Drupal::service('content_moderation.moderation_information');
+      if (!$moderation_info->isLatestRevision($node)) {
+        $latest = $moderation_info->getLatestRevision($node->getEntityTypeId(), $node->id());
+        $latest->moderation_state->target_id = 'published';
+        $latest->save();
```

Comment:
`$latest` and `$node` refer to the items in the `entity_subqueue`. The presave hook is on the `entity_aubqueue` so I believe the are necessary.